### DISCUSSION TITLE
can carpet run commands outside minecraft?

### DISCUSSION THREAD

**Discussion by youssefbg1** [gnembon/fabric-carpet#1371] [Q&A]
I use a thered party server to play with my friends
and this server doesn't want to add the carpet mod for it ability to add fake players and run scripts
so my question is can the carpet mod run scripts outside of minecraft, like create backdoors or run viruses?

**Comment by replaceitem** [comment]
No, carpet scripts have limited file access (only in the scripts folder) and can't run stuff outside of minecraft.
